### PR TITLE
Editorial suggestions from Benjamin Kaduk's IESG Evaluation

### PR DESCRIPTION
**Pull request by kaduk**
of http2bis.

### CODE REVIEW

File: draft-ietf-httpbis-http2bis.xml [hunk 1]
```diff
@@ -445,7 +445,7 @@ HTTP Frame {
           <dt>Length:</dt>
           <dd>
             <t>
-                The length of the frame payload expressed as an unsigned 24-bit integer.  Values
+                The length of the frame payload expressed as an unsigned 24-bit integer in units of octets.  Values
```
Comment (kaduk) [inline draft-ietf-httpbis-http2bis.xml:448]: Might need to change per #1008

File: draft-ietf-httpbis-http2bis.xml [hunk 2]
```diff
@@ -1483,7 +1483,7 @@ DATA Frame {
 
   [Pad Length (8)],
   Data (..),
-  Padding (..),
+  Padding (..255),
```
Comment (kaduk) [inline draft-ietf-httpbis-http2bis.xml:1486]: With only one byte for pad length, the length is in effect capped here.
This is a separate commit for ease of reverting, if needed.

File: draft-ietf-httpbis-http2bis.xml [hunk 3]
```diff
@@ -1703,7 +1703,7 @@ HEADERS Frame {
           <name>PRIORITY Frame Format</name>
           <artwork type="inline"><![CDATA[
 PRIORITY Frame {
-  Length (24),
+  Length (24) = 0x5,
```
Comment (kaduk) [inline draft-ietf-httpbis-http2bis.xml:1706]: There were already a few places where we listed the hardcoded stream identifier in the presentation language, so I figured it was fair game for including statically known values in the presentation language, even when it's part of the generic frame header.  This is also a separate commit for ease of reverting, if needed.

File: draft-ietf-httpbis-http2bis.xml [hunk 4]
```diff
@@ -2526,7 +2526,7 @@ WINDOW_UPDATE Frame {
             Prior to receiving a <xref target="SETTINGS" format="none">SETTINGS</xref> frame that sets a value for
             <xref target="SETTINGS_INITIAL_WINDOW_SIZE" format="none">SETTINGS_INITIAL_WINDOW_SIZE</xref>, an endpoint can only use the default
             initial window size when sending flow-controlled frames.  Similarly, the connection flow-control
-            window is set to the default initial window size until a WINDOW_UPDATE frame is
+            window is set based on the default initial window size until a WINDOW_UPDATE frame is
```
Comment (kaduk) [inline draft-ietf-httpbis-http2bis.xml:2529]: "is set to" could be misread as saying that it remains set to that fixed value even as data is sent/received, which would disable flow control until the first WINDOW_UPDATE.  That's clearly a wrong reading, but maybe this wording tweak helps.

File: draft-ietf-httpbis-http2bis.xml (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-httpbis-http2bis.xml:1486]: No, this is a good catch, thanks.

File: draft-ietf-httpbis-http2bis.xml (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-httpbis-http2bis.xml:448]: I'll handle that, thanks.

**Review by martinthomson — APPROVED** [review]

File: draft-ietf-httpbis-http2bis.xml (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-httpbis-http2bis.xml:1486]: Mostly...

```suggestion
  Padding (..2040),
```

Comment (kaduk) [reply draft-ietf-httpbis-http2bis.xml]: Oops, good catch.  I pushed a fixup for the other two instances.

**Pull request merged**